pypdf/annotations.py

```diff
@@ -82,27 +85,10 @@ def free_text(
         font_color: str = "000000",
         border_color: Optional[str] = "000000",
         background_color: Optional[str] = "ffffff",
-    ) -> DictionaryObject:
-        """
-        Add text in a rectangle to a page.
-
-        Args:
-            text: Text to be added
-            rect: array of four integers ``[xLL, yLL, xUR, yUR]``
-                specifying the clickable rectangular area
-            font: Name of the Font, e.g. 'Helvetica'
-            bold: Print the text in bold
-            italic: Print the text in italic
-            font_size: How big the text will be, e.g. '14pt'
-            font_color: Hex-string for the color, e.g. cdcdcd
-            border_color: Hex-string for the border color, e.g. cdcdcd.
-                Use ``None`` for no border.
-            background_color: Hex-string for the background of the annotation,
-                e.g. cdcdcd. Use ``None`` for transparent background.
+    ):
+        self[NameObject("/Subtype")] = NameObject("/Text")
```

Comment:
this should be /FreeText

Reply:
There are both, `Text` (12.5.6.4 Text Annotations) and `FreeText` (12.5.6.6 Free Text Annotations)

Reply:
agree but here the class is FreeText🤔

Reply:
Fixed it